src/libraries/Native/build-native.sh

```diff
@@ -69,6 +72,27 @@ fi
 if [[ "$__TargetOS" == OSX ]]; then
     # set default OSX deployment target
     __CMakeArgs="-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 $__CMakeArgs"
+elif [[ "$__TargetOS" == Android ]]; then
```

Comment:
```suggestion
elif [[ -z "$ROOTFS_DIR" && "$__TargetOS" == Android ]]; then
```
same on line 59, this will avoid conflicts with `eng/common/cross/toolchain.cmake`.

Reply:
Thanks, fixed.